x11-wm/page/page-1.9.9-r5.ebuild

```diff
@@ -31,7 +31,6 @@ DEPEND="${RDEPEND}
 
 src_prepare() {
 	default
-	eautoreconf
```

Comment:
1. why are you leaving `src_prepare` in then?
2. why have you not removed the `autotools` inherit?

Reply:
Hello,

I'm not used to ebuild, I will clean them up.

Thank you for you feedbacks :)